LoopKit/CarbKit/CarbStore.swift

```diff
@@ -351,7 +351,7 @@ public final class CarbStore: HealthKitSampleStore {
                 return
             }
 
-            self.handleUpdatedCarbData(updateSource: .queriedByHealthKit)
```

Comment:
[nit] Just curious...why was this removed?  Not a blocker just wondering...

Reply:
The update source wasn't being used anywhere, so just cleanup. I think it used to be used by uploading code, but that has changed so much.